populus/compilation/backends/viper.py

```diff
@@ -1,8 +1,38 @@
+import pprint
+import os
+
 from .base import (
     BaseCompilerBackend,
 )
 
 
 class ViperBackend(BaseCompilerBackend):
-    project_source_extensions = ('*.v.py', '*.vy')
-    test_source_extensions = ('*.py', )
+    project_source_glob = ('*.v.py', '*.vy')
+    test_source_glob = ('test_*.v.py', 'test_*.vy')
+
+    def get_compiled_contracts(self, source_file_paths, import_remappings):
+        try:
+            from viper import compiler
+        except ImportError:
+            raise Exception('viper needs to be installed to use ViperBackend as compiler backend.')
```

Comment:
I think using `ImportError` as the exception class here would be better than the bare base `Exception` class.